index.html

```diff
@@ -17,10 +17,7 @@ <h2>Who are the Scratch Forum Helpers?</h2>
 			<h2>What forums are the Forum Helpers most active in?</h2>
 			<p style="margin-left:200px;margin-right:200px">Generally we help out in the <a href="https://scratch.mit.edu/discuss/1/" class="FHULink">Suggestions</a>, <a href="https://scratch.mit.edu/discuss/3/" class="FHULink">Bugs And Glitches</a>, <a href="https://scratch.mit.edu/discuss/4/" class="FHULink">Questions About Scratch</a>, and <a href="https://scratch.mit.edu/discuss/7/" class="FHULink">Help With Scripts</a> Forums.
 			<h2>I want to be a Forum Helper. How do I apply?</h2>
-			<p style="margin-left:200px;margin-right:200px">We're glad that you're interested in applying! We are always on the lookout for new members! However, there is a requirement, which is that you must have at least 100 posts to apply. In order to apply, please go to the Forum Helpers Studio (There is a link in the header), and fill out the application listed in the description of the studio.</p>
-			<h2>I submitted my application! Now what?</h2>
-			<p style="margin-left:200px;margin-right:200px">Once you submit an application, three of the Studio Managers will look at your application and grade it out of 10 points. The scores will be added together. You must get at least 24 to join. If you don't get the number, you may reapply in a week!</p>
-			<br>
+			<p style="margin-left:200px;margin-right:200px">You don't have to apply! The Forum Helpers handpick awesome forum helpers and invite them to the official studio!
```

Comment:
Please change this to say "There isn't an application process, but if three managers agree that you are being active and helpful on the Scratch Discussion Forums, they may choose to invite you to become a Forum Helper."

Reply:
OK, I'll go ahead and change that up.

Reply:
Finished!